components/Search/Search.tsx

```diff
@@ -0,0 +1,47 @@
+import styles from "@/styles/components/Search.module.scss"
+import RCLink from "@/components/RCLink/RCLink"
+import { SearchBar } from "@nypl/design-system-react-components"
+
+/**
+ * The Search component renders and controls the Search form and
+ * advanced search link.
+ */
+const Search = () => {
+  return (
+    <section role="search" className={styles.searchContainer}>
```

Comment:
The DS' `SearchBar` component already adds a `role="search"` to the form element so this is redundant.

Reply:
Good catch, fixed.